Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -15,14 +15,19 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     bool isCharging() const;
 
     /*Return the time remaining to when the battery is charged or depleted*/
-    QTime timeWhenChargedOrDepleted() const;
+    QTime timeWhenChargedOrDepleted()const;
 
     /*Update Variables*/
     void addData(const BatteryData& batteryData);
 
 private:
     // Convention is to have an _ after private members
     const double initialStateOfChargePercent_;
-
+    double current;
+    double voltage;
+    QTime previousTime;
+    QTime currentTime;
+    double currentAmpHours;
+    double timeDifferenceStamp;
```

Comment:
Also, voltage, timeDifferenceStamp, and currentTime don't need to be private members. Their longevity is only one call in `addData`. The scope of all variables should be as small as possible.